modules/web/src/app/project-overview/template.html

```diff
@@ -34,69 +40,61 @@
                                 (refreshClusterTemplates)="clusterTemplatesChange.next()"
                                 (refreshBackups)="backupsChange.next()"></km-create-resource-panel>
     </mat-card-header>
-    <mat-card-content fxLayout="row wrap"
-                      fxLayoutAlign="space-between center"
-                      fxLayoutGap="10px grid">
-      <div fxLayoutAlign=" center"
-           fxLayoutGap="15px"
-           fxFlex="auto">
-        <i class="km-icon-mask km-icon-projects-card"></i>
-        <km-property>
-          <div key>Project</div>
-          <div value>{{project?.name}}</div>
-        </km-property>
-      </div>
+    <mat-card-content fxLayout="column wrap"
+                      fxLayoutGap="15px">
 
-      <div fxLayout="row wrap"
-           fxFlex="auto">
+      <div fxLayout="row wrap">
         <div fxLayoutAlign=" center"
              fxLayoutGap="15px"
-             fxFlex>
+             class="project-name-property">
+          <i class="km-icon-mask km-icon-projects-card"></i>
+          <km-property>
+            <div key>Project</div>
+            <div value>{{project?.name}}</div>
+          </km-property>
+        </div>
+        <div fxLayoutAlign=" center"
+             fxLayoutGap="15px">
           <i class="km-icon-mask km-icon-cluster"></i>
           <km-property>
             <div key>Clusters</div>
             <div value>{{clusters.length}}</div>
           </km-property>
         </div>
         <div fxLayoutAlign=" center"
-             fxLayoutGap="15px"
-             fxFlex>
+             fxLayoutGap="15px">
           <i class="km-icon-mask km-icon-cluster"></i>
           <km-property>
             <div key>External Clusters</div>
             <div value>{{externalClusters.length}}</div>
           </km-property>
         </div>
         <div fxLayoutAlign=" center"
-             fxLayoutGap="15px"
-             fxFlex>
+             fxLayoutGap="15px">
           <i class="km-icon-mask km-icon-cluster-template"></i>
           <km-property>
             <div key>Cluster Templates</div>
             <div value>{{clusterTemplates.length}}</div>
           </km-property>
         </div>
         <div fxLayoutAlign=" center"
-             fxLayoutGap="15px"
-             fxFlex>
+             fxLayoutGap="15px">
           <i class="km-icon-mask km-icon-backup"></i>
           <km-property>
             <div key>Backups</div>
             <div value>{{backups.length}}</div>
           </km-property>
         </div>
         <div fxLayoutAlign=" center"
-             fxLayoutGap="15px"
-             fxFlex>
+             fxLayoutGap="15px">
           <i class="km-icon-mask km-icon-key"></i>
           <km-property>
             <div key>SSH Keys</div>
             <div value>{{sshKeys.length}}</div>
           </km-property>
         </div>
         <div fxLayoutAlign=" center"
-             fxLayoutGap="15px"
-             fxFlex>
+             fxLayoutGap="15px">
```

Comment:
In order to fix the margin issue, we can remove `fxFlex="auto"` from parent `div` of all these stat elements. And to fix issue of text breaking into 2 lines (e.g. `External Clusters`), we should add `&nbsp;` instead of space (e.g. `External&nbsp;Clusters`)